providers/component-protocol/components/table/cell_builder.go

```diff
@@ -88,6 +88,14 @@ func NewTextCell(text string) ITypedCellBuilder {
 	return cb.typed()
 }
 
+// NewTextCellWithAll .
+func NewTextCellWithAll(text *commodel.Text) ITypedCellBuilder {
```

Comment:
Maybe no need this method.
Just use `func NewTextCell(text string, options ...commodel.TextOptions) ITypedCellBuilder {`

`NewTextCell("hello", commodel.WithTextEnableCopy)`

Reply:
Or `func NewText(text string, enableCopy ...bool)`